cpmonitor/views.py

```diff
@@ -506,6 +508,49 @@ def markdown_uploader_view(request):
     return HttpResponse(data, content_type="application/json")
 
 
+def mstr_view(request, municipality_key):
+    url = "https://www.marktstammdatenregister.de/MaStR/Einheit/EinheitJson/GetVerkleinerteOeffentlicheEinheitStromerzeugung"
+    params = {
+        "sort": "EinheitMeldeDatum-desc",
+        "page": 1,
+        "pageSize": 10000,
```

Comment:
Wouldn't it be better to implement proper paging? It might be better not to sent around huge chunks of data and right know it could be that we miss some pages.

Reply:
Yes, you're right. But this I would like to leave for the next version, I added a task to the issue.